source/mainnet/net/references/grpc.rst

```diff
@@ -54,45 +56,47 @@ Transactions
 
 .. _grpc-send-transaction:
 
-.. function:: SendTransaction(networkId, payload) -> Bool
+.. function:: SendTransaction(network_id: Int32, payload: [Byte]) -> Bool
 
    Send a transaction to the given network.
    The node will do basic transaction validation, such as signature checks and
    account nonce checks, and if these fail, the call will return a gRPC error.
    The payload is in binary encoding, read more in the
    :ref:`grpc-transaction-encoding` section.
 
-   :param Int32 networkId: The network that the transaction should be sent to
+   :param Int32 network_id: The network that the transaction should be sent to
                            (only ``100`` is currently supported).
    :param payload: Binary encoding of the transaction payload.
-   :type payload: |grpc-block-item|_
+   :type payload: ``[Byte]`` of |grpc-block-item|_
    :returns: Whether the transaction succeeded.
    :rtype: Bool
```

Comment:
"succeeded" is a somewhat ambiguous term. In fact, this returns `True` if the transaction was successfully submitted, and never returns `False`. If there was a problem with the transaction, a GRPC error code will be used:

- The `SendTransaction` function exposed via the gRPC interface now provides the caller with detailed error messages if the 
  transaction was rejected instead of just `False`. The function still returns `True` if 
  the transaction was accepted.
  The following gRPC error codes can be returned.
  - 'SUCCESS' The transaction was succesfully relayed to consensus.
  - 'INVALID_ARGUMENT' The transaction was deemed invalid or exceeds the maximum size allowed (the raw size of the transaction).
     In addition the error message contains information as to why the transaction was deemed invalid.
  - 'FAILED_PRECONDITION' The network was stopped due to an unrecognized protocol update.
  - 'DUPLICATE_ENTRY' The transaction was a duplicate.
  - 'INTERNAL' An internal error happened and as such the transaction could not be processed.
  The server will return a gRPC status if the transaction was deemed invalid.